data/governments/governmentForms.json

```diff
@@ -0,0 +1,28 @@
+{
+    "description": "List of different forms of government from wikipedia https://en.wikipedia.org/wiki/List_of_forms_of_government",
```

Comment:
Little typo:
```suggestion
    "description": "List of different forms of government from Wikipedia https://en.wikipedia.org/wiki/List_of_forms_of_government",
```

Reply:
Updated all of them to be lower case as well! Thank you for the feedback =)